src/utils/formatNumber.ts

```diff
@@ -114,19 +116,17 @@ export const formatAmountWithPrecision = (
 
 // Fiat formatting
 
-const getMinimumCurrencyDenominator = (number: string | number, currency: string): number => {
-  const float = Number(number)
-
+const getMinimumCurrencyDenominator = memoize((currency: string): number => {
   const formatter = new Intl.NumberFormat(undefined, {
     style: 'currency',
     currency,
   })
 
-  const fraction = formatter.formatToParts(float).find(({ type }) => type === 'fraction')
+  const fraction = formatter.formatToParts(1).find(({ type }) => type === 'fraction')
```

Comment:
would be easier to understand if
`fraction` -> `hasFraction`
or
`currencyHasFraction`

Reply:
This isn't a flag. We require `fraction.value.length` on line 130.